datastore/tests/integration/value_types_test.py

```diff
@@ -0,0 +1,23 @@
+"""Make sure all value types are serialized/deserialized correctly"""
```

Comment:
I think it's good to have tests that show that it actually works! I'm hoping that when there are more tests here, they can share most of their code. Pretty much everything except `props_insert = {'location': LatLng(49.2827, 123.1207)}`, in fact.